phoenix-core-server/src/main/java/org/apache/phoenix/cache/aggcache/SpillableGroupByCache.java

```diff
@@ -365,6 +366,10 @@ public void close() throws IOException {
                 }
             }
 
+            public boolean next(List<Cell> result, ScannerContext scannerContext) throws IOException {
+                return next(result);
```

Comment:
Why is the scannerContext being silently ignored here ?

Reply:
@tkhurana I am not sure what the implementation should be for scanners returned by SpillableGroupByCache and InMemoryGroupByCache